git-clone-repos.yaml

```diff
@@ -82,3 +90,43 @@
       with_together:
         - "{{ file_details.results }}"
         - "{{ REPOS }}"
+      when: NO_GIT_HOOKS != '1'
+
+    - name: Checking if pipenv package is installed
+      command: which pipenv
+      register: pipenv_check
+      ignore_errors: true
+      no_log: true
+      when: NO_PIPENV_SETUP != '1'
+    - name: Installing pipenv package
+      debug:
+        msg:
+          - " pipenv python packages is not installed"
+          - " Use command 'sudo pip3 install pipenv' to install pipenv"
+      when: NO_PIPENV_SETUP != '1' and pipenv_check.stdout == ""
+
+    - name: Checking for the file Pipfile
+      stat:
+        path: ~/{{ DIR_PATH }}/{{ item }}/Pipfile
```

Comment:
quotes?

Reply:
will make this change.
so using quotes is better practice?